frontend/app/components/templates/EkycPage/EkycPage.tsx

```diff
@@ -85,17 +121,85 @@ export const EkycPage = ({ serviceId, name, shortDesc, longDesc }: Props) => {
         )}
 
         {currentStep === 2 && (
-          <FaceLiveness
-            solutionId={serviceId}
-            nextStep={() => nextStep(3)}
-            setOpenModal={setOpenModal}
-            onArrival={() => createVisitorActivities(serviceId, session_id, 40)}
-            onChecking={() => createVisitorActivities(serviceId, session_id, 50)}
-            onResult={() => createVisitorActivities(serviceId, session_id, 60)}
-          />
+          <div>
+            <h3 className={styles.title}>Take A Selfie Photo</h3>
+            <Cam localkey="liveness_snapshot" nextStep={() => nextStep(3)} />
+          </div>
+        )}
+
+        {currentStep === 3 && (
+          <div>
+            <h3 className={styles.title}>KTP Photo</h3>
+            <Cam
+              localkey="ktp_snapshot"
+              nextStep={() => nextStep(4)}
+              videoConstraints={{ facingMode: { ideal: 'environment' } }}
+            />
+          </div>
         )}
 
-        {currentStep === 3 && <Feedback id={serviceId} onClick={() => setCurrentStep(1)}/>}
+        {currentStep === 4 && (
+          <div className={styles.result}>
+            <div className={styles.percentage}>
+              <h3 className={styles.title}>Liveness result</h3>
+              {!loading && result ? (
+                <>
+                  <span>{`${Math.trunc(
+                    result.face_liveness.liveness * 100
+                  )}%`}</span>
+                  <p>
+                    {result.face_liveness.live ? 'Verified' : 'Not Verified'}
+                  </p>
+                </>
+              ) : (
+                <Spinner />
+              )}
+            </div>
+
+            <div className={styles.percentage}>
+              <h3 className={styles.title}>Face Match Result</h3>
+              {!loading && result ? (
+                <>
+                  <span>{`${Math.trunc(
+                    result.face_match.similarity * 100
+                  )}%`}</span>
+                  <p>{result.face_match.match ? 'Verified' : 'Not Verified'}</p>
+                </>
+              ) : (
+                <Spinner />
+              )}
+            </div>
```

Comment:
Butuh handling kalau engga ada result (response status ok, tapi result kopong).
Tadi pas coba foto selfie liveness ok, foto ktp nya dikosongin eternal loading mutarrr". Kayanya bisa jadi TO DO buat phase selanjutnya (buat API handlernya juga)

Response body
![image](https://user-images.githubusercontent.com/52844949/140526191-b20cf3d4-c12e-44f2-8777-f569b83fd7e9.png)

Reply:
resolved on commit be48a1d